### ISSUE TITLE
[Accessibility] A11y-compliance / aria-selected is not allowed on role="presentation"

### ISSUE THREAD

**Issue by fspoettel** [salesforce-ux/design-system#312]
I'm currently using LDS in a react project with  eslint/`jsx-a11y` enabled. In the changelog, you state that "When selected, aria-selected="true" is placed on the parent li." This would give the following markup for selected items if I am not mistaken:

``` html
      <li class="slds-dropdown__item" role="presentation" aria-selected="true">
        <a href="javascript:void(0);" role="menuitem" tabindex="0">
          <span class="slds-truncate">Menu Item Four</span>
        </a>
      </li>
```

which throws ["the attribute aria-selected is not supported by the role presentation"](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/role-supports-aria-props.md). Is this project a11y-compliant and this a bug or should I stop linting via a11y?

Thank you!

**Comment by kaelig** [comment]
Thank you for reporting this and for mentioning the jsx-a11y plugin which I didn't know about but that we'll definitely test.

@SiTaggart is that an issue you're aware of?

**Comment by SiTaggart** [comment]
Yes, in fact `aria-selected` is not supported on menu items at all.

The good news is in the very latest release, 2.1.2 we fixed (hopefully) all of these menus.

Taken from the [release notes for 2.1.2](https://www.lightningdesignsystem.com/release-notes/):
> Corrected use of menuitemcheckbox for selectable menu items, with the use of aria-checked instead of aria-selected

So updating to the latest mark-up should stop you getting these warnings :)

Great use of `eslint/jsx-a11y` too! 👍

**Comment by fspoettel** [comment]
@SiTaggart Thanks for the quick follow-up, appreciated. I can see that the `Accessibility`-information on `Picklist` and `Menu` is up-to-date and correct. However, the information right under the Dropdown example [here](https://www.lightningdesignsystem.com/components/menus/) is out-of-date then as it would introduce the violation again? Sorry for the hassle!

**Comment by SiTaggart** [comment]
No hassle, glad someone is paying attention to the documentation. 

I'll reopen this, as the documentation is, as you noticed, out of sync now with the recommendation and HTML for the components.

**Comment by kaelig** [comment]
Thank you again for reporting this, @fspoettel. This just got fixed and the change will appear with the Summer ’17 release of the Design System.
